stan/math/fwd/mat/fun/quad_form_sym.hpp

```diff
@@ -0,0 +1,76 @@
+#ifndef STAN_MATH_FWD_MAT_FUN_QUAD_FORM_SYM_HPP
+#define STAN_MATH_FWD_MAT_FUN_QUAD_FORM_SYM_HPP
+
+#include <stan/math/fwd/core.hpp>
+#include <stan/math/fwd/mat/fun/multiply.hpp>
+#include <stan/math/fwd/mat/fun/dot_product.hpp>
+#include <stan/math/prim/mat/fun/quad_form_sym.hpp>
+
+namespace stan {
+  namespace math {
+
+    template<int RA, int CA, int RB, int CB, typename T>
+    inline Eigen::Matrix<stan::math::fvar<T>, CB, CB>
+    quad_form_sym(const Eigen::Matrix<stan::math::fvar<T>, RA, CA>& A,
+                  const Eigen::Matrix<double, RB, CB>& B) {
+      using stan::math::multiply;
```

Comment:
remove the `using` statement. Unnecessary now that everything is in the same namespace.